lib/matplotlib/text.py

```diff
@@ -1643,9 +1653,8 @@ def transform(renderer) -> Transform
 
             See :ref:`plotting-guide-annotation` for more details.
 
-            Defaults to 'data'.
-
-        textcoords : str, `.Artist`, `.Transform`, callable or tuple, optional
+        textcoords : str or `.Artist` or `.Transform` or callable or \
+(float, float), default: value of *xycoords*
```

Comment:
"default: xycoords"? (just above you write "default: xy", either way is fine but stay consistent)